### PR TITLE
Add first draft of mod 3 project

### LINKED ISSUE #117: Mod3 Project

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

File: module3/Projects/Mod3Project.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+## Project Description
+For this project, you will build the final iteration of the Message Logger we created in Mod1 and built on in Mod2. In Mod2 you added a database, and now you're going to add a frontend! You are encouraged to bring in code you wrote in the Mod2 project, especially for calculating message statistics.
+
+
+This project will give you the opportunity to showcase all of the concepts we have learned this Mod.  There might be skills that we covered all the way back in week 1, so don't forget to use those earlier lessons and labs as a resource!
+
+**Project Setup**
+To get started on this project:
+
+1. **fork and clone** this [starter repo](https://github.com/turingschool-examples/MvcMessageLogger)
```
Comment (memcmahon) [inline module3/Projects/Mod3Project.md:32]: I started up this repo with no problem 🎉 I also like that you called out places to put their deliverables :)

File: module3/Projects/Mod3Project.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+1. A way to see all users
+1. A way to add a new user
+
+A feature is not considered complete until it is tested, so make sure to test both of these features!
+
+Before writing code, there are two planning steps you need to complete and include here in the README.
+1. Draw a request/response diagram that includes all requests required to complete this iteration
+1. Draw a rough wireframe of the UI page(s) required to complete this iteration
+
+Add screenshots of your planning images in your project README, where it says:
```
Comment (memcmahon) [inline module3/Projects/Mod3Project.md:51]: Do you think they will add these artifacts through the github GUI, or in Visual Studio?  I'm anticipating some possible git merge conflicts.

File: module3/Projects/Mod3Project.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+#### REPLACE THIS HEADER WITH YOUR ITERATION 1 REQUEST/RESPONSE IMAGE
+#### REPLACE THIS HEADER WITH YOUR ITERATION 1 WIREFRAME
```
Comment (memcmahon) [inline module3/Projects/Mod3Project.md:53]: I think in the project kickoff we can introduce what wire-framing is - I don't think it is too complex of a topic.  (this is how the BE team introduces wireframing)

File: module3/Projects/Mod3Project.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+When you are finished with iteration 3, create a pull request and schedule time with a cohort-mate or your instructor for some code review.
+
+## Version Control Expectations
+In order to demonstrate your proficiency with version control, you will be using commits, branches and pull requests during this project.
+
+You must gather feedback from cohort-mates and/or instructors prior to merging any code into your `main` branch.
+
+You must provide feedback on at least one other student's pull request!
+
```
Comment (memcmahon) [inline module3/Projects/Mod3Project.md:121]: Should we include some expectations around usability?  Like, we should never have to type in an address, or use the back button?

**Review by memcmahon — APPROVED** [review]
Love it - just a couple small notes

File: module3/Projects/Mod3Project.md (hunk 2 above)
Comment (zoefarrell) [reply module3/Projects/Mod3Project.md:51]: Earlier in the mod we will have students install this extension to edit markdown in Visual Studio: https://marketplace.visualstudio.com/items?itemName=MadsKristensen.MarkdownEditor2

File: module3/Projects/Mod3Project.md (hunk 4 above)
Comment (zoefarrell) [reply module3/Projects/Mod3Project.md:121]: I like this idea

**Pull request merged**